File: docs/code-examples/all/point2d_simple.cpp
// Log some very simple points.

#include <rerun.hpp>

int main() {
    const auto rec = rerun::RecordingStream("rerun_example_points2d");
    rec.spawn().exit_on_failure();

    rec.log("points", rerun::Points2D({{0.0f, 0.0f}, {1.0f, 1.0f}}));

    // Log an extra rect to set the view bounds
    rec.log("bounds", rerun::Boxes2D::from_half_sizes({{2.0f, 1.5f}}));
}

File: docs/snippets/all/archetypes/point2d_simple.py
"""Log some very simple points."""

import rerun as rr
import rerun.blueprint as rrb

rr.init("rerun_example_points2d", spawn=True)

rr.log("points", rr.Points2D([[0, 0], [1, 1]]))

# Set view bounds:
rr.send_blueprint(rrb.Spatial2DView(visual_bounds=rrb.VisualBounds2D(x_range=[-1, 2], y_range=[-1, 2])))
